app/services/converters/base.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module Converters
+  # The parent class for the converters
+  # @abstract
+  class Base
+    CONTEXT = {
+      "dct": "http://purl.org/dc/terms/",
+      "desm": "http://desmsolutions.org/ns/",
+      "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+      "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+      "skos": "http://www.w3.org/2004/02/skos/core#"
+    }.freeze
+
+    DESM_NAMESPACE = URI("http://desmsolutions.org/ns/")
```

Comment:
As far as I understand, the namespace isn't expected to be a subject of frequent change (or any change, really), hence it's OK not to have it configurable.

Reply:
@excelsior  Yet, it does not seem to be a place to define it as a constant like this. It would be ok to define it in the environment configuration file instead, like `config/environments/development.rb`, for example.

I am aware of future features like different instances of this tool, so with that in mind, it would be nice to get it outside this file. Just my opinion.